pkg/jobrunaggregator/jobrunaggregatorlib/disruption.go

```diff
@@ -48,6 +49,44 @@ type BackendDisruption struct {
 	DisruptionMessages []string
 }
 
+// defined in origin/platformidentification/types.go
+// minimized here
+type ClusterData struct {
+	MasterNodesUpdated string
+}
+
+func GetMasterNodesUpdatedStatusFromClusterData(clusterData map[string]string) string {
+	// default is unknown
+	masterNodesUpdated := ""
+
+	// there can be multiple files (upgrade / conformance) if any of them indicate the master nodes updated
+	// we indicate that for the entire run
+	for _, clusterdataResults := range clusterData {
+		if len(clusterdataResults) == 0 {
+			continue
+		}
+
+		cd := &ClusterData{}
+		if err := json.Unmarshal([]byte(clusterdataResults), cd); err != nil {
```

Comment:
Should log the error here.

Reply:
So, I can but this is basically copy and paste from [right below](https://github.com/openshift/ci-tools/blob/6aa730873a19888644b4467373ec4ee13759f4d5/pkg/jobrunaggregator/jobrunaggregatorlib/disruption.go#L98-L100 ) .  I could update both places if we want.